secure_uninstall/models/module.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
+# Copyright 2014 David BEAL @ Akretion <david.beal@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+from openerp.exceptions import Warning as UserError
+from openerp.tools.config import config
+
+
+def _get_authorized_password():
+    """ You can define your own authorized keys
+    """
+    return [config.get("secure_uninstall")]
```

Comment:
Please could we have instead: [config.get("secure_uninstall"), config.get("admin_passwd")] ?

Rationale: using admin_passwd can be a security issue because admin_passwd has a lot of power and not all installations provide it (Odoo SA SaaS don't for instance), this is why a new password such as secure_uninstall is required indeed. But on the other hand, in an installation where we do provide an admin_passwd, then the user who know this password can already totally screw the installation if he wish, so I think we should also permit him to uninstall any module using this same admin_passwd without having to set up and communicate yet another password in this case.